AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyCommon.py

```diff
@@ -0,0 +1,46 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Base class for QtPy classes. Contains commonly used constants and generic behavior used on multiple QtPy classes
+"""
+
+import pyside_utils
+from PySide2 import QtWidgets, QtTest, QtCore
+
+
+class QtPyCommon:
+    """
+    """
+    def __init__(self):
+        self.var = None
+
+    def click_menu_option(self, window, option_text):
```

Comment:
You should be able to just use `pyside_utils.trigger_context_menu_entry`

Reply:
this function was written to click on menus in the upper left corner. I'm not sure the context menu function will work for that

Reply:
@cgalvan, I advised @michleza to improve the function name and docstring and re-request are review of your feedback.

Reply:
That makes sense, you could also use `pyside_utils.get_action_for_menu_path` that let's you specify sub-menus more explicitly but this works too.

Reply:
@michleza , as we move forward with this, I want to make sure we're not re-inventing the wheel.

We should look further into `pyside_utils` and possibly replace code in future test refactors/improvements as needed.  Even if it's just an aliased call to `pyside_utils` to give us that code hinting/completion in PyCharm.